clj-refactor.el

```diff
@@ -3653,7 +3661,7 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-create-fn-from-e
         (buffer-substring (point-min) (point-max)))
     s))
 
-(defun cljr--is-keyword? (s)
+(defun cljr--sans-ns-keyword? (s)
```

Comment:
I agree that the name better explains what the function does, but that's only because you found a bug in the function!  The intent everywhere else (than the new usage site added in this PR) is to check for keywords, aliased, fully-qualified, or not.